docs/guides/postgres/clustering/streaming_replication.md

```
@@ -55,347 +68,354 @@ spec:
+kubectl delete -n demo pg/ha-postgres pg/sync-postgres
+kubectl delete ns demo
 ```
 
 ## Next Steps
 
+- Learn about the [synchronous replication overview](/docs/guides/postgres/synchronous/index.md).
+- Run PostgreSQL with a [custom configuration file](/docs/guides/postgres/configuration/using-config-file.md).
 - Monitor your PostgreSQL database with KubeDB using [built-in Prometheus](/docs/guides/postgres/monitoring/using-builtin-prometheus.md).
 - Monitor your PostgreSQL database with KubeDB using [Prometheus operator](/docs/guides/postgres/monitoring/using-prometheus-operator.md).
 - Want to hack on KubeDB? Check our [contribution guidelines](/docs/CONTRIBUTING.md).
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)